diff -up kimageformats-5.99.0/src/imageformats/avif.cpp.omv~ kimageformats-5.99.0/src/imageformats/avif.cpp
--- kimageformats-5.99.0/src/imageformats/avif.cpp.omv~	2022-10-15 13:24:06.630785275 +0200
+++ kimageformats-5.99.0/src/imageformats/avif.cpp	2022-10-15 13:25:28.554290689 +0200
@@ -351,7 +351,7 @@ bool QAVIFHandler::decode_one_frame()
     rgb.rowBytes = result.bytesPerLine();
     rgb.pixels = result.bits();
 
-#if AVIF_VERSION >= 100101
+#ifdef AVIF_YUV_TO_RGB_DEFAULT
     // use faster decoding for animations
     avifResult res = avifImageYUVToRGB(m_decoder->image, &rgb, (m_decoder->imageCount > 1) ? AVIF_CHROMA_UPSAMPLING_NEAREST : AVIF_YUV_TO_RGB_DEFAULT);
 #else
@@ -782,7 +782,7 @@ bool QAVIFHandler::write(const QImage &i
             }
         }
 
-#if AVIF_VERSION >= 100101
+#ifdef AVIF_RGB_TO_YUV_DEFAULT
         res = avifImageRGBToYUV(avif, &rgb, AVIF_RGB_TO_YUV_DEFAULT);
 #else
         res = avifImageRGBToYUV(avif, &rgb);
